public/javascript/logout.js

```diff
@@ -11,8 +11,4 @@ async function logout() {
   }
```

Comment:
Here I switch the logout from a button to an anchor that doesn't have an event listener and goes to the logout route